assets/js/main.js

```diff
@@ -1,17 +1,10 @@
 var $window = $(window);
 var $htmlBody = $('html, body');
 
-var $logo = $('.logo h1');
 var $navigation = $('.header-nav-wrapper');
 var $navLinks = $navigation.find('.header-nav-list-item-link:not(.hire-us-link)');
-var $menuToggle = $('.menu-toggle');
 var $topBar = $('.top-bar');
-var $colorChangeElements = $navLinks.add($menuToggle).add($topBar);
-var $whoWeAreLink = $('.who-we-are-link');
-var $whatWeDoLink = $('.what-we-do-link');
-var $hireUsLink = $('.hire-us-link');
-var $hireUsButton = $('.tag-button');
-var $hireUsTriggers = $hireUsLink.add($hireUsButton);
+var $colorChangeElements = $navLinks.add($topBar);
 
 var buffer = 50;
```

Comment:
`setActiveNav`, `animateScroll`, and `getClickEvent` functions don't appear to be used anywhere either